github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -2358,3 +2358,80 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: false
```

Comment:
Don't we need `optional: true`?

Reply:
IIRC if `always_run: true` is set, this job is conditional anyway, so it is implicitly optional. Can't find the doc link for this right now, though.

Reply:
There it is: https://docs.prow.k8s.io/docs/legacy-snapshot/prow/jobs/#requiring-jobs-for-auto-merge-through-tide

So in short: if neither `run_if_changed` nor `skip_if_only_changed` are set this job is already conditional and doesn't match any set, so it's optional.